code/modules/art/statues.dm

```diff
@@ -573,23 +573,27 @@ Moving interrupts
 	var/static/list/plane_whitelist = list(FLOAT_PLANE, GAME_PLANE, FLOOR_PLANE)
 
 	/// Ideally we'd have knowledge what we're removing but i'd have to be done on target appearance retrieval
-	var/list/overlays_to_remove = list()
+	var/list/overlays_to_keep = list()
 	for(var/mutable_appearance/special_overlay as anything in content_ma.overlays)
 		var/mutable_appearance/real = new()
 		real.appearance = special_overlay
 		if(PLANE_TO_TRUE(real.plane) in plane_whitelist)
-			continue
-		overlays_to_remove += real
-	content_ma.overlays -= overlays_to_remove
+			content_ma.overlays -= real
+			overlays_to_keep += real
+		else
+			content_ma.overlays -= real
+	content_ma.overlays = overlays_to_keep
```

Comment:
```suggestion
			overlays_to_keep += real
	content_ma.overlays = overlays_to_keep
```
If we override overlays anyway couldn't we just do this?

Reply:
applies to underlays too for what it's worth

Reply:
I'll do that for both then